src/config/version.rs

```diff
@@ -17,7 +17,7 @@ impl VersionConfig {
 
     pub fn default_version(&self) -> &'static str {
         match self {
-            Self::Tailwind => "v4.1.4",
+            Self::Tailwind => "v4.1.8",
```

Comment:
I've been using the v4.1.10 version. Maybe that would be avoid a change after this one goes through.
```suggestion
            Self::Tailwind => "v4.1.10",
```